meetings/2024-01-17.md

```diff
@@ -0,0 +1,161 @@
+
+Proposed by: Maxime
+
+* MdJ: This is more a question than a PR maybe?
+* ML: Some implementation servers are not allowing multiple subjects in the payload when creating or updating resources. So, can't in some apps to apply the TypeIndex or any self-describing document. Was wondering it should not be part of the spec if / when we are accepting RDF document / graph we must accept the graph contains multiple subjects.
+* MdJ: The spec allows the server. So can never be sure. If a server only accepts triples about the document, I'd say that server is broken.
+* eP: Besides self-describing docs, certain predicates are used in a certain direction, like JSON-LD reverse. This is common in RDF. Since it is a graph, often things can be subject or object. I think it'd be interetsing to add a test case. What level would this requirement would be.
+* MdJ: Even if you do WAC for instance, you'd have different Authorization rules. A test would fail in that case. That server wouldn't be compliant.
+* PAC: Not sure I agree w/ you re any server should accept any type of content. I don't hav ea strong opinion. But can understand some people might want to develop servers maybe not generalist servers but still using some of the Solid server specification. Maybe it is not a good idea to rule out those people. I can share some things from the LDP work. I realised that some terms used in the group didn't make it to the spec. Used to talk about vanilla and chocolate servers - generalist servers and passive servers vs. the ones to put arbitrary resources.
+* eP: Should get back to this topic when server validation. If server advertises a shape, and not just arbitrary. I agree not every server can but can advertise the added payload.
+* MdJ: That'd be perhaps in future spec?
+* SC: Current spec says.. category of contraints, one imposed by the server, expectation that container should behave.. Anything beyond is open, spec allows the case PAC was mentioning. There's also the possibility to advertise those contraints. Existing servers, if they want to continue doing what they're doing might want to get into that habit. 
+* SC: There's a similar consideration raised in Solid Protocol, whether a read-only Solid server should be conforming. Normally we think read-write. Wehther a conforming server not implementing those methods.. Some servers might want to do the mminimal thing. That's fine. 
+* MdJ: "Advertising its chocolateness" (TM).
+* ML: So this constraint would be know at run-time and not when they're choosing their pod. So then I get a pod but not might as expected.
+* eP: Separate topic perhaps how people can choose. Who is setting the constraints. It wouldn't be arbitrary constraints. How we can educate users to picking a storage.
+* SC: Server could pass the test suite. When it's deployed it could completely reject requests including multiple subject. I don't think test suite will catch that. Comes down to whether URI owner allows that. 
```

Comment:
```suggestion
* ML: So this constraint would be known at run-time, and not when they're choosing their pod. So then I get a pod but might not be as expected.
* eP: Separate topic perhaps; how people can choose. Who is setting the constraints. It shouldn't be arbitrary constraints. How we can educate users picking a storage?
* SC: Server could pass the test suite. When it's deployed, it could completely reject requests including multiple subjects. I don't think the test suite will catch that. Comes down to whether URI owner allows that.
```